src/os/file/file.js

```diff
@@ -118,7 +118,11 @@ os.file.File.prototype.getContent = function() {
  * @param {?(ArrayBuffer|Object|string)} value
  */
 os.file.File.prototype.setContent = function(value) {
-  this.content_ = value;
+  if (value instanceof ArrayBuffer) {
+    this.content_ = os.file.normalizeWhitespace(new TextDecoder('utf-8').decode(value));
```

Comment:
You cannot assume UTF8 here. A better place for this call would be in `File.prototype.convertContentToString()`.